tests/tools/sessionvalidation/sessionvalidation.py

```diff
@@ -197,11 +197,6 @@ def validateSingleTransaction(txn):
                 _verbose_print("transaction request Host header doesn't have specified host")
                 retval = False
 
-            # reject if the host is localhost (since ATS seems to ignore remap rules for localhost requests)
-            if "127.0.0.1" in txn_req.getHeaders() or "localhost" in txn_req.getHeaders():
-                _verbose_print("transaction request Host is localhost, we must reject because ATS ignores remap rules for localhost requests")
-                retval = False
-
```

Comment:
This does not seem to have value, and it gets in the way of health checks that connect to the microserver at localhost directly.